itou/siae_evaluations/models.py

```diff
@@ -285,6 +285,26 @@ def get_email_eligible_siae(self):
         body = "siae_evaluations/email/eligible_siaes_body.txt"
         return get_email_message(to, context, subject, body)
 
+    @property
+    def state(self):
+
+        if any(
+            x
+            for x in self.evaluated_job_applications.all()
+            if x.state == evaluation_enums.EvaluatedJobApplicationsState.PENDING
+            or x.state == evaluation_enums.EvaluatedJobApplicationsState.PROCESSING
```

Comment:
pareil que l'autre PR, à toi de voir si tu fais un filter ou si tu itères en python. Faut compter les requetes.